ocaml/parsing/jane_syntax_parsing.ml

```diff
@@ -654,10 +660,35 @@ module Expression0 = Make_with_attribute (struct
   let location expr = expr.pexp_loc
   let with_location expr l = { expr with pexp_loc = l }
 
+  let fail_if_wrong_syntactic_category = true
+
   let attributes expr = expr.pexp_attributes
   let with_attributes expr pexp_attributes = { expr with pexp_attributes }
 end)
 
+(** Cases; embedded as [lhs guard -> [%%extension.EXTNAME] rhs] *)
+module Case0 = Make_with_extension_node (struct
+  type ast = case
+
+  let plural = "cases"
+  let location case = case.pc_rhs.pexp_loc
+  let with_location case l =
+    let pc_rhs = { case.pc_rhs with pexp_loc = l } in
+    { case with pc_rhs  }
+
+  let fail_if_wrong_syntactic_category = false
```

Comment:
This wants a comment explaining why it's set to `false` where every other one is set to `true`.

Reply:
It's possible that this should just reference the mli, if you end up including the commented I suggested for that.

Reply:
Added a comment referencing the previous one.